_getting-started/intro.md

```diff
@@ -0,0 +1,151 @@
+
+<img src="{{site.url}}{{site.baseurl}}/images/intro/cluster.png" alt="A cluster containing two indexes and two nodes" width="650">
+
+Despite being a piece of an OpenSearch index, each shard is actually a full Lucene index---confusing, we know. This detail is important, though, because each instance of Lucene is a running process that consumes CPU and memory. More shards is not necessarily better. Splitting a 400 GB index into 1,000 shards, for example, would place needless strain on your cluster. A good rule of thumb is to keep shard size as 10--50 GB.
+
+## Primary and replica shards
+
+In OpenSearch, shards may be _primary_ (the original) or _replica_ (a copy). By default, OpenSearch creates a replica shard for each primary shard. Thus, if you split your index into ten shards, OpenSearch creates ten replica shards. For example, consider a cluster described in the previous section. If you add one replica for each shard of each index in the cluster, your cluster will contain a total of 2 shards and 2 replicas for index 1 and 4 shards and 4 replicas for index 2, as shown in the following image. 
+
+<img src="{{site.url}}{{site.baseurl}}/images/intro/cluster-replicas.png" alt="A cluster containing two indexes with one replica shard for each shard in the index" width="700">
+
+These replica shards act as backups in the event of a node failure---OpenSearch distributes replica shards to different nodes than their corresponding primary shards---but they also improve the speed at which the cluster processes search requests. You might specify more than one replica per index for a search-heavy workload.
+
+## Advanced concepts
+
+The following section describes more advanced OpenSearch concepts.
+
+### Translog
+
+Any index changes, such as document indexing or deletion, are written to disk during a Lucene commit. However, Lucene commits are expensive operations, so they cannot be performed after every change to the index. Instead, each shard records every indexing operation in a transaction log called _translog_. When a document is indexed, it is added to the memory buffer and recorded in the translog. After a process or host restart, any data in the in-memory buffer is lost. Recording the document in the translog ensures durability because the translog is written to disk.
+
+Frequent refresh operations write the documents in the memory buffer to a segment and then clear the memory buffer. Periodically, a [flush](#flush) performs a Lucene commit, which includes writing the segments to disk using `fsync`, purging the old translog, and starting a new translog. Thus, a translog contains all operations that have not yet been flushed.
```

Comment:
I feel like this might be getting too detailed while muddying the key thing that users might need to know ("When is my data durable? When is my data searchable?").

I think we could say something like:
> An indexing or bulk call responds when the documents have been written to the translog and the translog is flushed to disk, so the updates are durable. The updates will be visible from search requests until after a _refresh_ operation (see below).

I almost feel like it would help to document these as steps in the lifecycle of an update, like:
1. An update is received by a primary shard and gets written to the shard's transaction log, which is flushed to disk (followed by an `fsync`) before the update is acknowledged. This guarantees durability.
2. The update is also passed to the Lucene index writer, which adds it to an in-memory buffer.
3. On refresh, the Lucene index writer flushes the in-memory buffers to disk (with each buffer becoming a new Lucene segment), and a new index reader is opened over the resulting segment files. The updates are now visible for search.
4. On a _flush_ operation, the shard `fsync`s the Lucene segments. Since the segment files are a durable representation of the updates, the translog is no longer needed to provide durability, do the updates can be purged from the translog.

If the OpenSearch process is terminated between the end of step 1 (when the update has been acknowledged) and the end of step 4 (when the updated Lucene segments have been flushed to disk), the updates will be replayed from the translog when the process restarts.

@smacrakis -- we talked briefly about this content. Is the above clearer or still too in-the-weeds?